playbooks/roles/integreatly/templates/rhmi_post_uninstall_housekeeping_extra_vars.yml.j2

```diff
@@ -0,0 +1,2 @@
+---
+cluster_name: '{{ '{{' }} cluster_id{{ '}}' }}'
```

Comment:
Should there be a space after `cluster_id` here?

Reply:
Not needed as space is here `{{ '}}`